src/main/java/frc/robot/Constants.java

```diff
@@ -55,6 +55,19 @@ public final class Constants {
     public static final double Y_PID_CONTROLLER_P = 0.2;
     public static final double THETA_PID_CONTROLLER_P = 8.0;
 
+    //photonvision constants, change all of these later
+    public static final double CAMERA_HEIGHT_METERS = Units.inchesToMeters(24); //change later
```

Comment:
If we're going to have this constant, we should also have the X and Y offsets for the transform in Vision.java